pkg/actors/actors.go

```diff
@@ -975,9 +997,13 @@ func (a *actorsRuntime) CreateReminder(ctx context.Context, req *CreateReminderR
 	a.reminders[req.ActorType] = reminders
 	a.remindersLock.Unlock()
 
-	err = a.startReminder(&reminder)
-	if err != nil {
-		return err
+	if ctx.Value("Test") != true {
```

Comment:
I really oppose putting in special code that is aware of a test that's being executed.

Reply:
Used a mock instead